ideascube/mediacenter/migrations/0013_escape_summary.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from django.db import migrations
+from django.utils.html import escape
+
+
+def escape_summary(app, schema_editor):
+    Document = app.get_model('mediacenter', 'Document')
+    db_alias = schema_editor.connection.alias
+    for document in Document.objects.using(db_alias).all():
+        document.summary = "<p>{}</p>".format(escape(document.summary))
```

Comment:
TinyMCE add a `<p>...</p>` by default around "blocks".
It can be disable with https://www.tinymce.com/docs/configure/content-filtering/#forced_root_block but, as said in the note : «Note that not using p elements as root block can severely cripple the functionality of the editor.»

I'm not sure how it will be cripple, but I prefer to do the same thing as tinyMCE would have be to not break anything.